primitives/xcm/src/origin_conversion.rs

```diff
@@ -82,3 +82,23 @@ where
 		})
 	}
 }
+
+/// Matches foreign assets from a given origin.
+/// Foreign assets are assets bridged from other consensus systems. i.e parents > 1.
+pub struct IsForeignConcreteAssetFrom<Origin>(PhantomData<Origin>);
```

Comment:
```suggestion
pub struct IsBridgedConcreteAssetFrom<Origin>(PhantomData<Origin>);
```

Please don't use the term Foreign asset, as we already use this term for assets taht come from the relay or other parachains